### PR TITLE
card time client

### PR DESCRIPTION
**Pull request by hyx930**
### PR types

Function optimization
### PR changes

OPs
### Describe

card time client

### CODE REVIEW

File: client/paddleflow/statistics/statistics_api.py [hunk 1]
```diff
@@ -113,3 +113,53 @@ def get_statistics_detail(cls, host, job_id: str, start=None, end=None,
             return False, data['message']
         statistics_job_detail_info = StatisticsJobDetailInfo.from_json(data)
         return True, statistics_job_detail_info
+
+    @classmethod
+    def get_statistics_by_name(cls, host, queue_name: str, start: str, end: str, run_id: str = None, header=None):
```
Comment (D0m021ng) [inline client/paddleflow/statistics/statistics_api.py]: get_cardtime_by_queue_name

File: client/paddleflow/statistics/statistics_api.py [hunk 2]
```diff
@@ -113,3 +113,53 @@ def get_statistics_detail(cls, host, job_id: str, start=None, end=None,
...
+        """
+        get statistics info, run_id is not supported yet
+        @param host: host url
+        @param queue_name: queue name
+        @param start: start time
+        @param end: end time
+        @param run_id: not supported yet
```
Comment (D0m021ng) [inline client/paddleflow/statistics/statistics_api.py]: remove it

File: client/paddleflow/statistics/statistics_api.py [hunk 3]
```diff
@@ -113,3 +113,53 @@ def get_statistics_detail(cls, host, job_id: str, start=None, end=None,
...
+        @param header: request header
+        @return: success: bool, resp: StatisticsQueueInfo
+        """
+        if not header:
+            raise PaddleFlowSDKException("InvalidRequest", "paddleflow should login first")
+
+        # TODO: support run_id
```
Comment (D0m021ng) [inline client/paddleflow/statistics/statistics_api.py]: remove it

File: client/paddleflow/statistics/statistics_api.py [hunk 4]
```diff
@@ -113,3 +113,53 @@ def get_statistics_detail(cls, host, job_id: str, start=None, end=None,
...
+        if run_id:
+            raise PaddleFlowSDKException("InvalidRequest", "run_id is not supported yet")
+
+        if len(queue_name) == 1:
+            pram = {
+                "runID": run_id,
+                "start": start,
```
Comment (D0m021ng) [inline client/paddleflow/statistics/statistics_api.py]: `start => startTime`

File: client/paddleflow/statistics/statistics_api.py [hunk 5]
```diff
@@ -113,3 +113,53 @@ def get_statistics_detail(cls, host, job_id: str, start=None, end=None,
...
+                "end": end,
```
Comment (D0m021ng) [inline client/paddleflow/statistics/statistics_api.py]: `end` -> `endTime`

**Review by D0m021ng — APPROVED** [review]

**Review by 8olio8 — APPROVED** [review]

**Pull request merged**